lenovo/legion/16irx9h/default.nix

```diff
@@ -0,0 +1,35 @@
+{
+  lib,
+  config,
+  ...
+}: {
+  imports = [
+    ../../../common/cpu/intel
+    ../../../common/gpu/nvidia/prime.nix
+    ../../../common/gpu/nvidia/ada-lovelace
+    ../../../common/pc/laptop
+    ../../../common/pc/laptop/ssd
+    ../../../common/hidpi.nix
+  ];
+
+  boot.initrd.kernelModules = ["nvidia"];
+  boot.extraModulePackages = [config.boot.kernelPackages.lenovo-legion-module config.boot.kernelPackages.nvidia_x11];
```

Comment:
the legion module is needed, but the rest should be already covered by the nvidia module.

Reply:
```suggestion
  boot.extraModulePackages = [config.boot.kernelPackages.lenovo-legion-module];
```